app-editors/efte/efte-1.1.recipe

```diff
@@ -0,0 +1,69 @@
+SUMMARY="A fork of FTE -- Folding Text Editor"
+DESCRIPTION="Advanced lightweight configurable editor eFTE is an advanced \
+programmers editor with goals of being lightweight, yet totally configurable.
+
+Support for user defined programming languages, menu systems and key bindings \
+are provided with many common defaults already defined.
+
+eFTE is still a new project, however, extended from the FTE editor which was
+first released in 1995, so eFTE is tried and true with many features for the
+programmer/text editor."
+HOMEPAGE="https://github.com/lanurmi/efte"
+COPYRIGHT="2008-2016 by eFTE SF Group.
+	1994-1998 by Marko Macek."
+LICENSE="Artistic
+	GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/lanurmi/efte/archive/refs/tags/efte-$portVersion.tar.gz"
+CHECKSUM_SHA256="6857ff5456e8b0bbfdfb2e524f339a6f38ad841b2b24cb385ec2af8a25e2e8ac"
+SOURCE_DIR="efte-efte-$portVersion"
+PATCHES="efte-$portVersion.patchset"
+
+ARCHITECTURES="x86_64 x86_gcc2"
```

Comment:
Any reason why not to use `all` architectures here?

Reply:
No need for secondary architecture, but it could be `ARCHITECTURES="all"` ? Other ports like riscv etc are not tested or build (at the moment), so it wouldn't be a big deal for now :)